manifests/config.pp

```diff
@@ -51,6 +51,7 @@
   #$prereg_price = 45,
   #$at_door_price = 60,
   $groups_enabled = true,
+  $group_discount - 10,
```

Comment:
noted that this typo was fixed right after in https://github.com/magfest/ubersystem-puppet/pull/89/files